Content.Server/Lathe/LatheSystem.cs

```diff
@@ -48,7 +51,7 @@ public override void Initialize()
 
             SubscribeLocalEvent<LatheComponent, BeforeActivatableUIOpenEvent>((u, c, _) => UpdateUserInterfaceState(u, c));
             SubscribeLocalEvent<LatheComponent, MaterialAmountChangedEvent>(OnMaterialAmountChanged);
-
+            SubscribeLocalEvent<LatheComponent, GotEmaggedEvent>(OnEmagged);
```

Comment:
handle this in `SharedLatheSystem`: the reason you have to do the popup yourself is because of prediction issues due to being server-side.